### ISSUE TITLE
Creating a collection with `flat/bq` + `cache=true` leads to infinite `prefill_progress` log messages

### ISSUE THREAD

**Issue by etiennedi** [weaviate/weaviate#6491]
### How to reproduce this bug?

```python
import weaviate
import weaviate.classes.config as wvcc

client = weaviate.connect_to_local()

# optionally, add to be able to re-run:
# client.collections.delete_all()

col = client.collections.create(
    "Test",
    vector_index_config=wvcc.Configure.VectorIndex.flat(
        quantizer=wvcc.Configure.VectorIndex.Quantizer.bq(cache=True)
    ),
)

```

### What is the expected behavior?

Not seeing an infinite number of logs

### What is the actual behavior?

<img width="2530" alt="Screenshot 2024-11-29 at 9 43 32 AM" src="https://github.com/user-attachments/assets/cd3bd854-6bdb-4e47-a301-46edb7c6b25f">

### Server Version

Tested on `v1.25.x`, likely all versions from that point on, but requires more investigation

### Weaviate Setup

Single Node

### Nodes count

1

### Code of Conduct

- [X] I have read and agree to the Weaviate's [Contributor Guide](https://weaviate.io/developers/contributor-guide) and [Code of Conduct](https://weaviate.io/service/code-of-conduct)

**Comment by etiennedi** [comment]
Additional info: The same behavior also occurs after a restart, so it looks like any empty collection that has a cache being prefilled creates those messages.

**Comment by etiennedi** [comment]
Fixed in  #6492, to be released in the next series of patches.
